src/CoreShop/Bundle/CoreBundle/CoreExtension/StoreValues.php

```diff
@@ -624,10 +624,12 @@ public function getDataForEditmode($data, $object = null, $params = [])
                 continue;
             }
 
+            $currency = $store->getCurrency();
+
             //Fill missing stores with empty values
             $storeData[$store->getId()] = [
                 'name' => $store->getName(),
-                'currencySymbol' => $store->getCurrency()->getSymbol(),
+                'currencySymbol' => $currency ? $currency->getSymbol() : '',
```

Comment:
We're on PHP 8, right? So why not:

```suggestion
                'currencySymbol' => $currency?->getSymbol() ?? '',
```